src/dom.js

```diff
@@ -0,0 +1,69 @@
+const hiddenClassName = 'hidden';
+
+/**
+ * Sets up the onclick handler for the call creation button.
+ * @param onclick
+ */
+export function setupCreateButton(onclick) {
+  const createBtn = document.getElementById('createRoom');
+  createBtn.onclick = () => {
+    onclick();
+  };
+}
+
+/**
+ * Adds a participant label to the presence list.
+ * @param label
+ */
+export function addToPresenceList(label) {
+  const participants = document.getElementById('participants');
+  const item = document.createElement('li');
+  item.innerText = label;
+  participants.appendChild(item);
+}
+
+export function showPresence() {
+  getPresenceEle().classList.remove('hidden');
```

Comment:
There are a couple spots where you use the `'hidden'` string instead of the variable you assigned it to. Can be swapped with `hiddenClassName`